tests/DocumentationExamplesTest.php

```diff
@@ -1540,6 +1540,85 @@ public function testCausalConsistency()
         ob_end_clean();
     }
 
+    /**
+     * @doesNotPerformAssertions
+     */
+    public function testWithTransactionExample()
+    {
+        $this->skipIfTransactionsAreNotSupported();
+
+        $uriString = static::getUri(true);
+
+        // phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly
+        // Start Transactions withTxn API Example 1
+        /*
+         * For a replica set, include the replica set name and a seedlist of the members in the URI string; e.g.
+         * uriString = 'mongodb://mongodb0.example.com:27017,mongodb1.example.com:27017/admin?replicaSet=myRepl'
+         * For a sharded cluster, connect to the mongos instances; e.g.
+         * uriString = 'mongodb://mongos0.example.com:27017,mongos1.example.com:27017/admin'
+         */
+
+        $client = new \MongoDB\Client($uriString);
+
+        // Prerequisite: Create collections. CRUD operations in transactions must be on existing collections.
+        $client->selectCollection(
+            'test',
+            'foo',
```

Comment:
The examples in the DRIVERS ticket use "mydb1" and "mydb2" for each collection, rather than "test" for both. I'm not sure if that's pertinent to demonstrate a transaction across two databases, but I think we should what's specified.